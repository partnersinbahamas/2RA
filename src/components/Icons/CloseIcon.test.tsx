import React from 'react';
import { render } from '@testing-library/react';

import CloseIcon from './CloseIcon';

describe('CloseIcon', () => {
  it('renders small', () => {
    const { container } = render(<CloseIcon size="SMALL" />);
    expect(container).toMatchSnapshot();
  });
  it('renders medium', () => {
    const { container } = render(<CloseIcon size="MEDIUM" />);
    expect(container).toMatchSnapshot();
  });
  it('renders large', () => {
    const { container } = render(<CloseIcon size="LARGE" />);
    expect(container).toMatchSnapshot();
  });
  it('renders default', () => {
    const { container } = render(<CloseIcon />);
    expect(container).toMatchSnapshot();
  });
});
