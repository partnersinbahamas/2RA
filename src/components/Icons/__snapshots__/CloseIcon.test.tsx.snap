// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon renders default 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="5.5 5.5 21 21"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M7.6181 6.194L16.0001 14.576L24.2931 6.3C24.3836 6.20443 24.4928 6.12856 24.6139 6.07711C24.7351 6.02566 24.8655 5.99975 24.9971 6.001C25.5471 6.001 26.0001 6.421 26.0001 7.004C26.0001 7.264 25.9351 7.457 25.7101 7.684L17.4231 16L25.7081 24.317C25.8896 24.4974 25.9939 24.7412 25.9991 24.997C26.0011 25.1292 25.9766 25.2604 25.927 25.383C25.8773 25.5055 25.8036 25.6168 25.7102 25.7104C25.6167 25.8039 25.5055 25.8777 25.383 25.9275C25.2605 25.9772 25.1293 26.0019 24.9971 26C24.7371 26 24.5441 25.935 24.3171 25.71L16.0001 17.423L7.7071 25.7C7.61662 25.7956 7.5074 25.8714 7.38627 25.9229C7.26514 25.9743 7.1347 26.0002 7.0031 25.999C6.87091 26.001 6.73966 25.9765 6.61712 25.9269C6.49459 25.8772 6.38326 25.8035 6.28973 25.7101C6.1962 25.6166 6.12237 25.5054 6.07262 25.3829C6.02287 25.2604 5.99821 25.1292 6.0001 24.997C6.0001 24.737 6.0651 24.544 6.2901 24.317L14.5771 16L6.2911 7.683C6.10971 7.50256 6.0054 7.25879 6.0001 7.003C6.0001 6.453 6.4201 6 7.0031 6C7.1971 6 7.4231 6.065 7.6181 6.194Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`CloseIcon renders large 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="5.5 5.5 21 21"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M7.6181 6.194L16.0001 14.576L24.2931 6.3C24.3836 6.20443 24.4928 6.12856 24.6139 6.07711C24.7351 6.02566 24.8655 5.99975 24.9971 6.001C25.5471 6.001 26.0001 6.421 26.0001 7.004C26.0001 7.264 25.9351 7.457 25.7101 7.684L17.4231 16L25.7081 24.317C25.8896 24.4974 25.9939 24.7412 25.9991 24.997C26.0011 25.1292 25.9766 25.2604 25.927 25.383C25.8773 25.5055 25.8036 25.6168 25.7102 25.7104C25.6167 25.8039 25.5055 25.8777 25.383 25.9275C25.2605 25.9772 25.1293 26.0019 24.9971 26C24.7371 26 24.5441 25.935 24.3171 25.71L16.0001 17.423L7.7071 25.7C7.61662 25.7956 7.5074 25.8714 7.38627 25.9229C7.26514 25.9743 7.1347 26.0002 7.0031 25.999C6.87091 26.001 6.73966 25.9765 6.61712 25.9269C6.49459 25.8772 6.38326 25.8035 6.28973 25.7101C6.1962 25.6166 6.12237 25.5054 6.07262 25.3829C6.02287 25.2604 5.99821 25.1292 6.0001 24.997C6.0001 24.737 6.0651 24.544 6.2901 24.317L14.5771 16L6.2911 7.683C6.10971 7.50256 6.0054 7.25879 6.0001 7.003C6.0001 6.453 6.4201 6 7.0031 6C7.1971 6 7.4231 6.065 7.6181 6.194Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`CloseIcon renders medium 1`] = `
<div>
  <svg
    class="sc-dmyCSP dRcSrL"
    fill="currentcolor"
    viewBox="4.75 4.75 14.5 14.5"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M5.00021 18C5.00021 17.719 5.09421 17.5 5.28121 17.312L10.5942 12L5.28021 6.68703C5.11603 6.49616 5.02745 6.25176 5.03121 6.00003C5.03121 5.43703 5.46921 5.03103 6.00021 5.03103C6.28121 5.03103 6.50021 5.12503 6.68821 5.28103L12.0002 10.594L17.3132 5.28103C17.4019 5.18905 17.5088 5.11655 17.6271 5.06817C17.7454 5.01979 17.8725 4.99658 18.0002 5.00003C18.5632 5.00003 19.0002 5.43803 19.0002 6.00003C19.0002 6.28103 18.9062 6.50003 18.6872 6.68803L13.4072 12L18.6872 17.313C18.7891 17.3954 18.8705 17.5003 18.9248 17.6196C18.9791 17.7388 19.0049 17.8691 19.0002 18C19.0029 18.1295 18.9798 18.2582 18.9325 18.3787C18.8851 18.4992 18.8143 18.6091 18.7242 18.7021C18.6341 18.795 18.5265 18.8692 18.4075 18.9204C18.2886 18.9715 18.1607 18.9986 18.0312 19C17.8958 19.0043 17.7612 18.9784 17.637 18.9242C17.5129 18.8701 17.4023 18.7891 17.3132 18.687L12.0002 13.407L6.6872 18.687C6.59826 18.789 6.48783 18.8699 6.36384 18.924C6.23986 18.9781 6.10542 19.0041 5.9702 19C5.84065 18.9987 5.71264 18.9718 5.59358 18.9207C5.47452 18.8696 5.36677 18.7954 5.27656 18.7024C5.18636 18.6094 5.11549 18.4995 5.06805 18.3789C5.02061 18.2583 4.99755 18.1296 5.00021 18Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`CloseIcon renders small 1`] = `
<div>
  <svg
    class="sc-guDLey kBBjNb"
    fill="currentcolor"
    viewBox="3.75 3.75 12.75 12.75"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M4.00039 15C3.99668 14.8675 4.01965 14.7355 4.06792 14.612C4.11619 14.4885 4.18878 14.3759 4.28139 14.281L8.56339 10L4.28039 5.68701C4.09971 5.50416 3.999 5.25706 4.00039 5.00001C4.00039 4.50001 4.40639 4.00001 5.00039 4.00001C5.25039 4.00001 5.50039 4.09401 5.68839 4.28101L10.0004 8.56301L14.3134 4.28101C14.4961 4.09997 14.7432 3.99889 15.0004 4.00001C15.5314 4.00001 16.0004 4.43801 16.0004 5.00001C16.0004 5.25001 15.9064 5.50001 15.7194 5.68801L11.4384 10L15.7184 14.281C15.8112 14.3758 15.8839 14.4884 15.9324 14.6119C15.9808 14.7354 16.004 14.8674 16.0004 15C16.0004 15.531 15.5624 16 15.0004 16C14.8678 16.0037 14.7359 15.9807 14.6124 15.9325C14.4889 15.8842 14.3763 15.8116 14.2814 15.719L10.0004 11.406L5.71939 15.72C5.62439 15.8124 5.51179 15.8848 5.38829 15.9329C5.26478 15.981 5.13287 16.0038 5.00039 16C4.43739 16 4.00039 15.531 4.00039 15Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
